proto/connectrpc/conformance/v1/config.proto

```diff
@@ -32,42 +32,61 @@ message Config {
   repeated ConfigCase exclude_cases = 3;
 }
 
+// Features define the feature set that a client or server support. They are
+// used to determine the tests that are possible to be run in a client-server
+// combination. They are defined in YAML files and are specified as part of the
+// --conf flag to the test runner.
 message Features {
   // TODO: we could probably model some of the constraints on what are valid vs.
   //       invalid (i.e. conflicting/impossible) features using protovalidate rules
 
+  // Supported HTTP versions.
   // If empty, HTTP 1.1 and HTTP/2 are assumed.
   repeated HTTPVersion versions = 1;
+  // Supported protocols.
   // If empty, all three are assumed: Connect, gRPC, and gRPC-Web.
   repeated Protocol protocols = 2;
+  // Supported codecs.
   // If empty, "proto" and "json" are assumed.
   repeated Codec codecs = 3;
+  // Supported compression algorithms.
   // If empty, "identity" and "gzip" are assumed.
   repeated Compression compressions = 4;
+  // Supported stream types.
   // If empty, all stream types are assumed. This is usually for
   // clients, since some client environments may not be able to
   // support certain kinds of streaming operations, especially
   // bidirectional streams.
   repeated StreamType stream_types = 5;
+  // Whether H2C (unencrypted, non-TLS HTTP/2 over cleartext) is supported.
   // If absent, true is assumed.
   optional bool supports_h2c = 6;
+  // Whether TLS is supported.
   // If absent, true is assumed.
   optional bool supports_tls = 7;
+  // Whether the client supports TLS certificates.
   // If absent, false is assumed. This should not be set if
   // supports_tls is false.
   optional bool supports_tls_client_certs = 8;
+  // Whether trailers are supported.
   // If absent, true is assumed. If false, implies that gRPC protocol is not allowed.
   optional bool supports_trailers = 9;
+  // Whether half duplex bidi streams are supported over HTTP/1.1.
   // If absent, false is assumed.
   optional bool supports_half_duplex_bidi_over_http1 = 10;
+  // Whether Connect via GET is supported.
   // If absent, true is assumed.
   optional bool supports_connect_get = 11;
+  // Whether the Connect version header (Connect-Protocol-Version) is required.
```

Comment:
Does this field also apply to the Connect version query param? If so, should we consider renaming?

Reply:
I don't think we actually use it at all. I had some ideas early on (when I added this field) about writing tests that verified this -- like a test case that would omit it and be expected to fail if this was set to true (and another case, for when this was false, that would omit and expect to succeed). But they never to came to pass.

We should probably instead remove it entirely. It might be nice to add it as another check to the reference server -- so even though it's technically optional in the protocol spec, we'd verify that conformant client implementations always include it (to maximize inter-op, just in case a server is configured to require it).

Reply:
Ah yeah, this isn't used either. Cool to add a TODO to the server checks?

Reply:
Certainly!

Reply:
Actually in looking at it. It's not set anywhere in a config YAML, but it does indirectly get used in `internal/app/connectconformance/config.go`. Not sure if all of that should be ripped or not?

Reply:
Yes, it should be ripped out. But a TODO is fine as well.

Reply:
Mentioned below. Going to put this in its own PR.